docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/transforming-enriched-data/reusable/batch-only/_index.mdx

```diff
@@ -18,5 +18,19 @@
     <td><code>monitoring.sentry.dsn</code></td>
     <td>Optional. For tracking runtime exceptions.</td>
 </tr>
-
-
+<tr>
```

Comment:
Were these settings added in 5.1? We should state so

Reply:
Only `deduplication.natural` is added in 5.1. Other settings under `deduplication` were already in there. I forgot to include them during the refactoring of transformer docs.